### PR TITLE
Fix a few RCT1 build height discrepancies

### LINKED ISSUE #20094: Classic wooden coaster should be 34.5 meters, not 60 meters
### Operating System

Windows 11, 64-Bits

### OpenRCT2 build

OpenRCT2, v0.4.4-209-gd464e71 (d464e71 on develop) provided by GitHub

### Base game

RollerCoaster Tycoon 2

### Area(s) with this issue?

_No response_

### Describe the issue

so, in rct1, you can build the wooden coaster (classic wooden coaster) at a maximum of 34,5 meters high [115 feet] (23 units) above the ground, but in OpenRCT2, the classic wooden coaster have the same height as the rct2 wooden coaster, which is 60 meters [200 feets] (34 units) above the ground, which dosen't make sense, especially if the chainlift is only between 4 to 8 kmph (3 to 5mph)

### Steps to reproduce

1. open a park whit the classic wooden coaster
2. build yourself a classic wooden coaster
3. try to build it as high as you can
4. you get stopped by "supports too high" at around 60 meters, which is the bug (rct1 only have a maximum of 34,5 meters (traducted in rct 2 metric)), why should I get to build them higher than I could do it in rct1?

### Attachments

what it is like right now (notice that the hybrid is also here but it shouldn't be too much borthered):
Parc sans nom 2023-04-29 19-03-24

how it should be :
Parc sans nom 2023-04-29 19-03-42

### PR DESCRIPTION
**Pull request by ZeeMaji**
Fixes a few rides to have max build height parity with RCT1, nearly all coasters should now have the same max build heights as RCT1 or higher than RCT1.
This also fixes #20094 by changing the classic woodie's max build height to it's correct value of 24.
There were a few oddities however.

1: The corkscrew coaster/hypercoaster & twister/hyper-twister coasters are the same ride type in RCT1 and thus, share the same build height. I believe in the past it has been said that the corkscrew & twister coasters will not have RCT1 build height parity due to how absurdly high the RCT1 build heights are for these ride type.
2: The hyper-twister in RCT1 does not appear to have had a height limit, as i could build it all the way up to the height limit of the game which is 61 units, i have set the max build height of the hyper-twister to this as this should emulate the RCT1 build height for it without outright removing it's height limit.
3: The stand-up coaster has a max build height of 30 in RCT1, however as discussed in #18171 the RCT1 stand-up coaster will likely be implemented as it's own ride type in the future so i have left the RCT2 stand-up's build height unchanged.
4: The inverted coaster's build height is 1 value higher than it's listed height number in RCT1, i believe this is due to the track being taller due to it being an inverted track type.

Test park
[buildheights.SV4.tar.gz](https://github.com/OpenRCT2/OpenRCT2/files/11362791/buildheights.SV4.tar.gz)

### CODE REVIEW

**Review by Gymnasiast — APPROVED** [review]
Could probably do with a changelog entry (under the Change category, I would say).

File: src/openrct2/ride/coaster/meta/ClassicWoodenRollerCoaster.h [hunk 1]
```diff
@@ -35,7 +35,7 @@ constexpr const RideTypeDescriptor ClassicWoodenRollerCoasterRTD =
    SET_FIELD(NameConvention, { RideComponentType::Train, RideComponentType::Track, RideComponentType::Station }),
    SET_FIELD(EnumName, nameof(RIDE_TYPE_CLASSIC_WOODEN_ROLLER_COASTER)),
    SET_FIELD(AvailableBreakdowns, (1 << BREAKDOWN_SAFETY_CUT_OUT) | (1 << BREAKDOWN_RESTRAINTS_STUCK_CLOSED) | (1 << BREAKDOWN_RESTRAINTS_STUCK_OPEN) | (1 << BREAKDOWN_VEHICLE_MALFUNCTION) | (1 << BREAKDOWN_BRAKES_FAILURE)),
-   SET_FIELD(Heights, { 41, 24, 8, 11, }),
+   SET_FIELD(Heights, { 24, 24, 8, 11, }),
```
Comment (Broxzier) [inline src/openrct2/ride/coaster/meta/ClassicWoodenRollerCoaster.h:38]: Since this is now lower and by quite a bit, players will have less freedom on how high to build? A track design that could be placed before, now wouldn't be possible to be changed at the highest points.

Comment (ZeeMaji) [reply src/openrct2/ride/coaster/meta/ClassicWoodenRollerCoaster.h:38]: This is the original height the classic wooden coaster should have as it was in RCT1 and keep in mind this ride type is meant to be a port of the RCT1 wooden coaster to openrct2, the height it currently has is an unintentional leftover from when it was being created from the RCT2 wooden coaster. 
Any designs made with RCT1 itself should still all be placeable, this would only affect classic wooden coaster designs made in openRCT2 between now and the ride type being introduced in last october.
Other similar changes have been made to it and the similarly ported classic mini coaster since their reintroductions in openrct2 to correct other accuracy differences that still would have affected track designs made with openrct2, notably the max lift speed being lowered on the classic woodie recently, and the ride ratings being change for both of them to better reflect their RCT1 values.

**Comment by ZehMatt** [comment]
Quite frankly not a fan of restricting the users with arbitrary limits, can you summarize what the implications of the change are and to why we need them? Just because RCT1 did it this way is not really a good justification in my opinion

**Comment by Gymnasiast** [comment]
@ZehMatt The ride type in question only exists for RCT1 compat - it wasn’t in RCT2. As such, making its stats in line with RCT1 will bring back some balance in those scenarios, avoiding an unintentionally overpowered ride type. It essentially fixes something I simply forgot when I added the Classic Wooden RC last year.

**Comment by ZehMatt** [comment]
I'm not really against changing it if my concerns fall flat, it doesn't seem to have existed that long so it's probably fine. As for breaking changes, just because it happened before doesn't mean it has to happen again, if we constantly change the mechanics then people will be inclined to not play it anymore as it can become frustrating.

**Review by Broxzier — APPROVED** [review]
